src/devices/Vl53L0X/SquadInfo.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Device.Vl53L0X
+{
+    internal class SpadInfo
```

Comment:
What's spad? https://en.wikipedia.org/wiki/Single-photon_avalanche_diode ?

Reply:
Yes as defined line 749 in the comments for the ```GetSpadInfo()``` function